### PR TITLE
Updates the biographical collection note layout

### LINKED ISSUE #718: Clear division is needed when there are bioghist notes for multiple creators
When there are two (or more) creator agents attached to a resource record there needs to be a clear division between the notes for each creator. See for example: https://findingaids-beta.princeton.edu/catalog/C0292#description. 

# Technical implementation

* Update our aspace plugin (https://github.com/pulibrary/princeton_ead_exporter) to export bioghistnotes with links to the agents that they're for.
* Index notes alongside the user they're for in some sort of JSON structure for display.
* Display the user alongside the note.

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #718 by implementing an updated design for the collection show pages:

image

### CODE REVIEW

File: spec/requests/catalog_request_spec.rb [hunk 1]
```diff
@@ -108,6 +108,15 @@
     end
   end
 
+  describe "collection-level bioghist notes" do
+    xit "renders the bioghist notes on the collection show page with <hr />" do
```
Comment (jrgriffiniii) [inline spec/requests/catalog_request_spec.rb:112]: This should not be failing (I am not certain what is triggering this, but should the last CircleCI build pass, I will remove this).

**Review by tpendragon — CHANGES_REQUESTED** [review]
This looks good, but we'll have to wait until Lyrasis installs our updated plugin in aspace-staging and then refresh all the fixtures (delete spec/fixtures/aspace/generated and run `rake pulfalight:fixtures:refresh_aspace_fixtures` then fix any test failures.) I assume you manually copied in C0292, but we need to avoid not being able to refresh the fixtures as much as possible.

**Comment by tpendragon** [comment]
@jrgriffiniii Can you update the rest of the EADs as well? Or at least those that haven't changed drastically (to the point irrelevant tests won't pass)

**Comment by jrgriffiniii** [comment]
I am sorry for the delay with this, I am currently experiencing the following recurring error:

```bash
I, [2021-09-02T10:15:07.798679 #75113]  INFO -- : Regenerated AC259
rake aborted!
MultiXml::ParseError: 96:6604686: FATAL: Premature end of data in tag c line 96
[...]/pulfalight/app/services/aspace_fixture_generator.rb:179:in `get_content'
```

**Comment by tpendragon** [comment]
What's after AC259? That looks like maybe an EAD with ruby errors in it is coming out?

**Comment by jrgriffiniii** [comment]
I'm going to try and restart my environment, I'm receiving this:

```bash
$ curl -H "X-ArchivesSpace-Session: $SESSION" "https://aspace-staging.princeton.edu/staff/api/repositories/3/resource_descriptions/1769"
{"error":"Sinatra::NotFound"}
```

These IDs were parsed by debugging through the Ruby code base.

**Comment by jrgriffiniii** [comment]
I apologize, I am still having this problem, it appears to just affect `MC016`:

```
170: def fixture_files
    171:   @fixture_files ||=
    172:     begin
    173:       #ead_ids.lazy.map do |eadid|
    174:       ['MC016'].lazy.map do |eadid|
    175:         binding.pry
[1] pry(#<AspaceFixtureGenerator>)> eadid
=> "MC016"
[2] pry(#<AspaceFixtureGenerator>)> uri, repo_code = client.ead_url_for_eadid(eadid: eadid)&.first
=> ["repositories/3/resource_descriptions/1769", "publicpolicy"]
[3] pry(#<AspaceFixtureGenerator>)> uri, repo_code = client.ead_url_for_eadid(eadid: eadid)
=> {"repositories/3/resource_descriptions/1769"=>"publicpolicy"}
```

The `client` here is configured for the staging environment:

```
=> #<Aspace::Client:0x00007f87f282cfb8
 @config=
  #<ArchivesSpace::Configuration:0x00007f87f29e80c8
   @base_repo="",
   @base_uri="https://aspace-staging.princeton.edu/staff/api",
```

...and this cannot be resolved from the staging server:

```
[STAGING_HOST]:~$ curl -H "X-ArchivesSpace-Session: $SESSION" "https://aspace-staging.princeton.edu/staff/api/repositories/3/resource_descriptions/1769"
{"error":"Sinatra::NotFound"}
```

**Comment by jrgriffiniii** [comment]
While I have forcibly-published this within the staging environment, this does not seem to have assisted:

image

I will try and see about fixing the failing test before returning to this.

**Comment by jrgriffiniii** [comment]
https://github.com/pulibrary/pulfalight/compare/issues-718-jrgriffiniii-biohist-wip?expand=1 just has the fixtures for C0292 replaced by using the GUI to download the affected ArchivesSpace Resources manually. This is passing on CircleCI: https://app.circleci.com/pipelines/github/pulibrary/pulfalight?branch=issues-718-jrgriffiniii-biohist-wip

**Comment by jrgriffiniii** [comment]
Deprecated by https://github.com/pulibrary/pulfalight/pull/895

**Pull request closed without merging**